oci/spec_opts_linux.go

```diff
@@ -171,7 +171,7 @@ func escapeAndCombineArgs(args []string) string {
 }
 
 // WithCDI updates OCI spec with CDI content
-func WithCDI(annotations map[string]string, cdiSpecDirs []string) SpecOpts {
```

Comment:
If it is related to CRI only, please consider to make it in the `pkg/cri/opts` package.
Or, `WithCDI`'s interface can be `WithCDI(annos, registry)`.

Reply:
@fuweid Thank you for the review. Yes, it's CRI only. Moved it to pkg/cri/opts as you've suggested.